x-pack/plugins/security_solution/public/detections/components/rules/rule_preview/use_preview_route.tsx

```diff
@@ -103,68 +60,30 @@ export const usePreviewRoute = ({
 
   useEffect(() => {
     clearPreview();
```

Comment:
Is this supposed to clear the preview page when the steps are updated?

Reply:
Good catch, we don't need this. We only clear the preview when user manually triggers refresh.